pkg/fips/fips.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright © 2025 SUSE LLC
+SPDX-License-Identifier: Apache-2.0
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fips
+
+import (
+	"context"
+
+	"github.com/suse/elemental/v3/pkg/chroot"
+	"github.com/suse/elemental/v3/pkg/sys"
+)
+
+func Enable(ctx context.Context, s *sys.System) error {
+	stdOut, err := s.Runner().RunContext(ctx, "/usr/bin/fips-mode-setup", "--enable", "--no-bootcfg")
```

Comment:
I believe this is decommissioned [upstream](https://gitlab.com/redhat-crypto/fedora-crypto-policies/-/merge_requests/205#9ac97f30b729f8d1c92231a3f266fc15fbf8c8c6).

We are a couple of commits behind on [SLFO:Main](https://build.suse.de/projects/SUSE:SLFO:Main/packages/crypto-policies/files/crypto-policies.spec?expand=1) hence why the script is still present.

Reply:
It also changes cryptopolicies. calling the command is still the recommended way for SL16, even if upstream decided to change things.